### PR TITLE
Bugfix of "Add reason to medkit"

### PR DESCRIPTION
**Pull request by Emojigit**
Sorry, my PRs have a lot of problems before. here is the list that this PR fix:
 * Fix reasons while attack and damage
 * Remove `die` reason, add `attack` reason

### CODE REVIEW

**Comment by Emojigit** [comment]
@Panquesito7

File: mods/pvp/medkits/init.lua [hunk 1]
```diff
@@ -52,15 +52,15 @@ end
...
+	if reason == "attack" then
+		return " because you attacked other player!"
 	elseif reason == "move" then
 		return " because you moved!"
 	elseif reason == "die" then
 		return " because you died!"
 	else
 		return "!"
-	end
+	end
```
Comment (LoneWolfHT) [inline mods/pvp/medkits/init.lua:63]: Trailing whitespace

**Review by LoneWolfHT — CHANGES_REQUESTED** [review]

File: mods/pvp/medkits/init.lua [hunk 2]
```diff
@@ -52,15 +52,15 @@ end
 -- move: move more then 1m
 -- die: die because any reason
 local function reason_handler(reason)
-	if reason == "damage" then
-		return " because someone damaged you!"
+	if reason == "attack" then
+		return " because you attacked other player!"
 	elseif reason == "move" then
 		return " because you moved!"
 	elseif reason == "die" then
```
Comment (Panquesito7) [inline mods/pvp/medkits/init.lua:59]: I still see die reason?

File: mods/pvp/medkits/init.lua (hunk 1 above)
Comment (Emojigit) [reply mods/pvp/medkits/init.lua:63]: I will add this. But why?

**Review by Panquesito7 — APPROVED** [review]

**Comment by Panquesito7** [comment]
Just need to fix [LuaCheck](https://github.com/MT-CTF/capturetheflag/pull/734/checks?check_run_id=1653483292) warnings **before** merge.

**Comment by Emojigit** [comment]
@Panquesito7 I don't know what does the warning means.
```
mods/pvp/medkits/init.lua:63:5: line contains trailing whitespace
```

File: mods/pvp/medkits/init.lua [hunk 3]
```diff
@@ -50,17 +50,17 @@ end
...
 	elseif reason == "move" then
 		return " because you moved!"
-	elseif reason == "die" then
-		return " because you died!"
+	elseif reason == "damage" then
+		return " because someone damaged you!"
 	else
 		return "!"
-	end
+	end
```
Comment (Panquesito7) [inline mods/pvp/medkits/init.lua:63]:
```suggestion
	end
```

**Review by Panquesito7 — APPROVED** [review]
@LoneWolfHT can be merged. 👍

File: mods/pvp/medkits/init.lua (hunk 3 above)
Comment (Emojigit) [reply mods/pvp/medkits/init.lua]: @Panquesito7 Resolved

**Pull request merged**